### DISCUSSION TITLE
If sd3 comes out, can fooocus upgrade directly?

### DISCUSSION THREAD

**Discussion by 812913329** [lllyasviel/Fooocus#2739] [General]
Or is it too involved that I have to start a new project or fooocus will no longer be updated and upgraded?

**Comment by ILikeToasters** [comment]
Support for SD3 will need to be added which may take time if they even decide to add support for it. Much will depend on how much needs to be changed to support it.

**Comment by mashb1t** [comment]
Fooocus currently is in a stable state with little to no bugs in the code. Feature development is paused as lllyasviel is focusing on other projects such as Forge + (a) private project(s) and I have a lot going on non-open source related in my spare time .

- lllyasviel already confirmed that he and the team will be back when the next version of SD is released, see https://github.com/lllyasviel/Fooocus/discussions/2154#discussioncomment-8389770.
- Package updates are currently paused, features are fully developed and you can find most of them in [my fork](https://github.com/mashb1t/Fooocus) (which btw is now over 350 commits ahead of Fooocus).

Hope this answers your question.

**Reply by ILikeToasters** [reply]
I am using your fork more and more these days. Like some of the added stuff. I tend to only use the main one when making videos for YouTube now.
